targets/intel-nuc.nix

```diff
@@ -27,16 +27,48 @@
           ../modules/graphics/weston.nix
 
           formatModule
+
+          {
+            boot.kernelParams = [
+              "intel_iommu=on,igx_off,sm_on"
+              "iommu=pt"
+
+              # Passthrough Intel WiFi card
+              "vfio-pci.ids=8086:a0f0"
+            ];
+          }
         ]
         ++ extraModules;
     };
     netvm = "netvm-${name}-${variant}";
   in {
     inherit hostConfiguration netvm;
     name = "${name}-${variant}";
-    netvmConfiguration = import ../microvmConfigurations/netvm {
-      inherit nixpkgs microvm system;
-    };
+    netvmConfiguration =
+      (import ../microvmConfigurations/netvm {
+        inherit nixpkgs microvm system;
+      })
+      .extendModules {
+        modules = [
+          {
+            microvm.devices = [
+              {
+                bus = "pci";
+                path = "0000:00:14.3";
+              }
+            ];
+
+            # For WLAN firmwares
+            hardware.enableRedistributableFirmware = true;
+
+            networking.wireless = {
+              enable = true;
+
+              # networks."SSID_OF_NETWORK".psk = "WPA_PASSWORD";
```

Comment:
SSID / password we could also refactor like authentication.nix - would also support .gitignore for developer local secrets until we develop proper way to support these.